blocks/library/heading/index.js

```diff
@@ -117,7 +126,9 @@ export const settings = {
 							subscript: level,
 						} ) )
 					}
-				/>
+				>
+				{ alignmentToolbar }
```

Comment:
Can we tab this once more? (Ideally we should have an ESLint rule for this)

Reply:
Fixed